basins.py

```diff
@@ -2,13 +2,32 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import regionmask
+import xarray as xr
 
+# basins codes used:
+
+ind_SO = 1   # Southern Ocean
+ind_ATL = 2  # Atlantic Ocean
+ind_PAC = 3  # Pacific Ocean
+ind_ARC = 4  # Arctic Ocean
+ind_IND = 5  # Indian Ocean
+ind_MED = 6  # Mediterranean Sea
+ind_BLK = 7  # Black Sea
+ind_HUD = 8  # Hudson Bay
+ind_BAL = 9  # Baltic Sea
+ind_RED = 10 # Red Sea
+ind_PER = 11 # Persian Gulf
```

Comment:
The CMIP6/CF standard (Griffins et al. 2016) does not have a separate code for the Persian Gulf, although we’ve historically kept one in OM4 development.  We need an option to add the Persian Gulf points to the Indian basin if a user requests it.